odl/discr/discr_ops.py

```diff
@@ -372,15 +372,15 @@ def _call(self, x, out=None):
 
         Parameters
         ----------
-        x : ``domain`` element
+        x : ``domain`` `element`
```

Comment:
I tried to find all places, perhaps I missed some. Hopefully we don't get clashes with the `element` method. I'll check that.